### ISSUE TITLE
style

### ISSUE THREAD

**Issue by gertcuykens** [bendavis78/paper-date-picker#27]
![image](https://cloud.githubusercontent.com/assets/863731/10593032/90aacd1a-76c0-11e5-9f2a-4681f6f26415.png)

I think style has changed a little?

https://www.polymer-project.org/1.0/docs/devguide/styling.html

```css
<!-- import the module  -->
<link rel="import" href="../shared-styles/shared-styles.html">
<dom-module id="x-foo">
  <template>
    <!-- include the style module by name -->
    <style include="shared-styles"></style>
```

https://github.com/bendavis78/paper-date-picker/blob/master/paper-date-picker.html#L74-L76

**Comment by gedw99** [comment]
+1

**Comment by bendavis78** [comment]
Fixed in 1.1.0
